.github/workflows/build-azure-functions-packages.yaml

```diff
@@ -0,0 +1,65 @@
+name: Build and Install GSF Packages on Sample Azure Functions
+
+on:
+  push:
+    branches: [ dev, main ]
+  pull_request:
+    branches: [ dev, main ]
+    paths:
+      - 'src/**'
+      - '.github/workflows/**'
+      - 'samples/azure-function/**'
+
+env:
+  EMISSIONS_DOCKERFILE_PATH: samples/azure-function/emissions-azure-function/Dockerfile
+  FORECAST_DOCKERFILE_PATH: samples/azure-function/forecast-azure-function/Dockerfile
+  EM_CONTAINER_IMAGE_NAME: em-runnable-container
+  EM_CONTAINER_RUNTIME_NAME: em-az-func
+  FC_CONTAINER_IMAGE_NAME: fc-runnable-container
+  FC_CONTAINER_RUNTIME_NAME: fc-az-func
+
+jobs:
+  container_emissions_azure_function:
```

Comment:
A matrix may be a bit difficult for forecast since we don't send a single string but build up the url with headers and a post body. May be more worth it for all the functions that have a single url hit to eventually be in a strategy

Reply:
Shifted to all be in one so not necessary